src/tests/Endpoint.spec.ts

```diff
@@ -1,64 +1,129 @@
-        expect(getEndpoint()).toBe('https://platform.cloud.coveo.com');
-    });
-
-    describe('development environment', () => {
-        it('should return the development US endpoint when no region is specified', () => {
-            expect(getEndpoint(Environment.dev)).toBe('https://platformdev.cloud.coveo.com');
+    describe('platform endpoint', () => {
+        it('should return the prod US endpoint by default', () => {
+            expect(getEndpoint()).toBe('https://platform.cloud.coveo.com');
         });
 
-        it('should not add a region suffix for the US region', () => {
-            expect(getEndpoint(Environment.dev, Region.US)).toBe('https://platformdev.cloud.coveo.com');
-        });
+        describe('development environment', () => {
+            it('should return the development US endpoint when no region is specified', () => {
+                expect(getEndpoint(Environment.dev)).toBe('https://platformdev.cloud.coveo.com');
+            });
 
-        it('should return the endpoint corresponding to each specified environment and region', () => {
-            expect(getEndpoint(Environment.dev, Region.EU)).toBe('https://platformdev-eu.cloud.coveo.com');
-            expect(getEndpoint(Environment.dev, Region.AU)).toBe('https://platformdev-au.cloud.coveo.com');
+            it('should not add a region suffix for the US region', () => {
+                expect(getEndpoint(Environment.dev, Region.US)).toBe('https://platformdev.cloud.coveo.com');
+            });
+
+            it('should return the endpoint corresponding to each specified environment and region', () => {
+                expect(getEndpoint(Environment.dev, Region.EU)).toBe('https://platformdev-eu.cloud.coveo.com');
+                expect(getEndpoint(Environment.dev, Region.AU)).toBe('https://platformdev-au.cloud.coveo.com');
+            });
         });
-    });
 
-    describe('staging environment', () => {
-        it('should return the development US endpoint when no region is specified', () => {
-            expect(getEndpoint(Environment.staging)).toBe('https://platformqa.cloud.coveo.com');
+        describe('staging environment', () => {
+            it('should return the development US endpoint when no region is specified', () => {
+                expect(getEndpoint(Environment.staging)).toBe('https://platformqa.cloud.coveo.com');
+            });
+
+            it('should not add a region suffix for the US region', () => {
+                expect(getEndpoint(Environment.staging, Region.US)).toBe('https://platformqa.cloud.coveo.com');
+            });
+
+            it('should return the endpoint corresponding to each specified environment and region', () => {
+                expect(getEndpoint(Environment.staging, Region.EU)).toBe('https://platformqa-eu.cloud.coveo.com');
+                expect(getEndpoint(Environment.staging, Region.AU)).toBe('https://platformqa-au.cloud.coveo.com');
```

Comment:
QA-AU does not exist IIRC xD

Reply:
No, but this test is for `platformqa-au` ;)

Reply:
No matter if it really exists or not, the platform-client needs to support it. The day we add it in the backend, I'm pretty sure we won't go in the platform-client to add the test.

Reply:
Ok good :+1: